src/apphosting/secrets/index.ts

```diff
@@ -215,25 +215,33 @@ export async function fetchSecrets(
  * to ensure the exported secrets accurately reflect the chosen environment.
  *
  * @param allAppHostingYamlPaths An array of paths to all available `apphosting.*.yaml` files.
- * @returns A Promise that resolves to an `AppHostingYamlConfig` object representing the combined
- *          configuration to be used for exporting configurations.
  */
 export async function getConfigToExport(
```

Comment:
I wonder if this should be a helper method in config.ts, ie

GetConfigForEnvironment(...);

This type of logic seems broadly useful for non-secret related workflows